src/doc/trpl/unsafe.md

```diff
@@ -49,32 +49,16 @@ and mutable (`&mut`) references have some aliasing and freezing
 guarantees, required for memory safety.
 
 In particular, if you have an `&T` reference, then the `T` must not be
-modified through that reference or any other reference. There are some
+modified through that reference or any other reference or pointer and must not
+alias with any `&mut` reference. There are some
 standard library types, e.g. `Cell` and `RefCell`, that provide inner
```

Comment:
it'd be nice to wrap this whole paragraph rather than have the weird line break here